### PR TITLE
fixes #1452 modifications to ensure altimetry moving average / step consideration

### LINKED ISSUE #1452: Altitude and profile calculation
On a souvent échangé sur ce sujet et des choses ont été faites mais nous venons de constater à nouveau un calcul assez décalé de la réalite. 

En effet la randonnée LA CABANE DU PRE D'ANTONI (http://rando.ecrins-parcnational.fr/fr/la-cabane-du-pre-dantoni) affiche un dénivelé positif de 870 mêtres.
Pourtant elle part de 1471m et culmine à 1855m soit une différence de 384m.

Et ce dénivelé correspond bien plus à la réalité.

Est-ce l'effet "dent de scie" aussi répercuté dans le profil, qui entraîne ce cumule de mini-dénivelés positifs ?

profile

Discuté ici - https://github.com/makinacorpus/Geotrek/issues/776 et normalement réglé avec https://github.com/makinacorpus/Geotrek/issues/840

### PR DESCRIPTION
**Pull request by submarcos**

### CODE REVIEW

**Comment by leplatrem** [comment]
That's a lot of new code! If I were you I would write some tests. Think of future maintainers !

**Comment by camillemonchicourt** [comment]
Yes, it is a crucial part of Geotrek. 
Also a few more comments would be useful to make code easier to read and maintain and for me to understand what is done. 
Thanks.

File: geotrek/altimetry/sql/10_utilities.sql [hunk 1]
```diff
@@ -142,3 +219,147 @@ BEGIN
     RETURN result;
 END;
 $$ LANGUAGE plpgsql;
+
+CREATE OR REPLACE FUNCTION geotrek.ft_elevation_infos(geom geometry, seuil integer) RETURNS elevation_infos AS $$
```
Comment (leplatrem) [inline geotrek/altimetry/sql/10_utilities.sql]: nit: seuil is `threshold` in english

**Comment by leplatrem** [comment]
The code is pretty clear and well commented. 

However I wonder how it was possible to write it without tests. Using browser refresh ?

**Pull request merged**